src/hotspot/share/opto/opaquenode.hpp

```diff
@@ -26,8 +26,9 @@
 #define SHARE_OPTO_OPAQUENODE_HPP
 
 #include "opto/node.hpp"
-#include "opto/opcodes.hpp"
-#include "subnode.hpp"
+#include "opto/subnode.hpp"
+
```

Comment:
Noticed that `opcodes.hpp` was unused and `subnode.hpp` missed the `opto` prefix. Fixed here as well.